src/ducks/components/intents/InstallAppIntent.jsx

```diff
@@ -83,6 +84,9 @@ export class InstallAppIntent extends Component {
       null
     )
     const error = !!errorKey && new Error(t(errorKey))
+    const isReadyWithoutErrors = isReady && !error
+    const doConfiguration =
```

Comment:
Repeated code, should be in a function imho. Btw `do` prefix is much a function name prefix. For a variable or a function returning a boolean maybe `hasToBeConfigured` or `isNeedingConfiguration` is better, even a bit longer. What do you think ?